gobbler/views/main.html

```diff
@@ -1,5 +1,13 @@
 <template name="main">
```

Comment:
Can we move away from the template 'main'? I feel like it's not adequately descriptive. Maybe we should go with GibletManager or something? (Both in terms of file name and template name). Thoughts?